working/augmentations/feature-specification.md

```diff
@@ -65,35 +65,22 @@ adding *augmentations*.
 
 ### Augmentation declarations
 
-This feature introduces the modifier `augment` as the first token of many
-kinds of declarations. These declarations are known as *augmentation
-declarations*.
+A declaration marked with the modifier `augment` creates an *augmentation
+declaration* (or just "augmentation"). In contrast, an unmarked declaration is
+an *introductory declaration*.
 
-*In Dart without this feature there are no augmentation declarations. Now
-that we are adding augmentation declarations we need to have a term that
-denotes a declaration which is not an augmentation. That is, it is one of
-the "normal" declarations that we've had all the time.*
+An augmentation declaration doesn't create a new entity in the program. Instead,
+it adds to some existing introductory declaration with the same name.
+Augmentation declarations can modify types, functions, members, and almost any
+other kind of declaration in Dart. They can add new members to, add new values
+to enums, or to the `with` or `implements` clauses, fill in function bodies,
+etc.
 
-We say that a declaration which is not an augmentation declaration is an
-*introductory declaration*.
-
-Augmentation declarations include:
-
-*   Type augmentations, which can add new members to types, add new values to
-    enums, or add to the `with` or `implements` clauses.
-
-*   Function augmentations, which can provide a body.
-
-These operations cannot be expressed today using only imports, exports, or
-part files. Any Dart file (library file or part file) can contain
-augmentation declarations. *In particular, an augmentation can augment a
-declaration in the same file in which it occurs.*
-
-An augmentation can fill in a body for a declared member that has no body, but
-can't *replace* an existing body or add to it. In order to allow augmentations
-to provide bodies for static methods and top-level functions, we allow
-declarations of those to be "abstract" and lack a body as long as a body is
-eventually provided by an augmenting declaration.
+An augmentation can be in the same file as the introductory declaration it
+applies to, or in separate part files, but they must both be in the same
+library. Augmentations allow authoring a library in multiple separate pieces or
+files (some of which are likely generated) in a flexible manner, but they aren't
+for injecting new behavior into libraries outside of the author's control.
```

Comment:
... outside of the library author's control ...

(The person trying to inject into a library is also an author.)